composer.json

```diff
@@ -15,6 +15,10 @@
     "php": "^7.2 || ^8.0",
     "ext-json": "*"
   },
+  "suggest": {
+    "ext-imagick": "*",
+    "ext-gd": "*"
```

Comment:
I think we should consider adding a reason for why these are suggested.

Reply:
Is there a way to provide a reason?

Reply:
I added them because PhpStorm was complaining about them not being in `composer.json`. Granted, it still complains, although less loudly, that they are just suggested and not required.

Reply:
As per docs, yes - https://getcomposer.org/doc/04-schema.md#suggest

Reply:
Oh, interesting:

![image](https://github.com/WordPress/performance/assets/134745/765564ad-bac7-4ded-a250-3b8187c41668)

I guess you can't suggest specific versions of things then?

Reply:
How's this?

```suggestion
    "ext-imagick": "Required to use Modern Image Format's Dominant_Color_Image_Editor_Imagick class",
    "ext-gd": "Required to use Modern Image Format's Dominant_Color_Image_Editor_GD class"
```

Reply:
Much better. I think we can tweak the version info in the text itself. Something like:
```
"ext-imagick": "Required at least {version constraint} to use Modern Image Format's Dominant_Color_Image_Editor_Imagick class",
```

Reply:
What I meant is that the freeform nature of `suggest` means that composer wouldn't be able to install the suggested dependencies automatically.